### ISSUE TITLE
Workspace inheritance not working with Dioxus CLI

### ISSUE THREAD

**Issue by marcerhans** [DioxusLabs/dioxus#1367]
**Problem**
When running "dx build" or "dx serve" it fails because I use workspace fields in the package section. It works fine with good old "cargo run".

```
I get the following error:
RUST_BACKTRACE=1 dx serve
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Parse(Error { inner: ErrorInner { kind: Custom, line: Some(23), col: 10, at: Some(518), message: "invalid type: map, expected a sequence", key: ["package", "authors"] } })', /home/marcus/.cargo/registry/src/github.com-1ecc6299db9ec823/dioxus-cli-0.4.0/src/config.rs:253:67
stack backtrace:
   0: rust_begin_unwind
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:64:14
   2: core::result::unwrap_failed
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/result.rs:1791:5
   3: dioxus_cli::config::CrateConfig::new
   4: dioxus_cli::cli::serve::Serve::serve::{{closure}}
   5: tokio::runtime::park::CachedParkThread::block_on
   6: tokio::runtime::scheduler::multi_thread::MultiThread::block_on
   7: tokio::runtime::runtime::Runtime::block_on
   8: dx::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Steps To Reproduce**

1. Have the following in your Cargo.toml
```toml
[workspace.package]
authors = ["marcerhans"]

[package]
authors = { workspace = true }
```

2. Run "dx build" or "dx serve".

**Expected behavior**
I expected the project to build and run (serve).

**Environment:**
 - Dioxus version: 0.4.0
 - Rust version: 1.71.1
 - OS info: Fedora 37
 - App platform: desktop

**Comment by Demonthos** [comment]
We need to [update cargo_tomu](https://github.com/DioxusLabs/dioxus/blob/master/packages/cli/Cargo.toml#L25) to parse the new cargo workspace options

**Comment by tigerros** [comment]
Fixed in #1471
